client/my-sites/stats/stats-purchase/index.tsx

```diff
@@ -130,35 +130,39 @@ const StatsPurchasePage = ( {
 			/>
 			<div className="stats">
 				<QueryProductsList type="jetpack" />
-				{
-					// TODO: if the page is commercial and already has a commercial plan we can either redirect them or display a message
-				 }
 				{ isLoading && (
 					<div className="stats-purchase-page__loader">
 						<LoadingEllipsis />
 					</div>
 				) }
 				{ ! isLoading && (
 					<>
-						{ ( isFreeOwned || isCommercialOwned || isPWYWOwned ) && (
+						{ isCommercialOwned && (
+							<div className="stats-purchase-page__notice">
+								<StatsPurchaseNotice siteSlug={ siteSlug } />
```

Comment:
Would 'StatsPurchase`d`Notice' sounds better?

Reply:
After a second thought, I think keeping the original `StatsPurchaseNotice` would be best. The other files start with `stats-purchase-<component>`. I used the same convention here.
If you prefer the the name in a past tense, I can change it, but it would stand out from other files in the directory.